core/callable_bind.cpp

```diff
@@ -0,0 +1,163 @@
+#include "callable_bind.h"
```

Comment:
Missing copyright header.

Reply:
And review spacing between functions (yes, I'll go pester clang-format devs about it again ;))